pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -398,7 +398,8 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 		}
 
 		// InterPodAffinity Predicate
-		status = podAffinityFilter.PreFilter(context.TODO(), state, task.Pod)
+		// TODO use framework.PreFilterResult
```

Comment:
Do you want to add some thing more here? if not, please delete useless codes or comments.